modules/artifact-activation/src/main/java/org/mule/runtime/module/artifact/activation/api/plugin/PluginPatchesResolver.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.module.artifact.activation.api.plugin;
+
+import org.mule.runtime.module.artifact.activation.internal.plugin.DefaultPluginPatchesResolver;
+import org.mule.tools.api.classloader.model.ArtifactCoordinates;
+
+import java.net.URL;
+import java.util.List;
+
+/**
+ * Resolves the patches that must be applied to a given plugin.
+ *
+ * @since 4.5
+ */
+public interface PluginPatchesResolver {
+
+  /**
+   * @return the default implementation of a {@link PluginPatchesResolver}.
```

Comment:
what does default mean in this context? It it a no-op of what standalone mule does? Please clarify/rename